docs_src/data_block.ipynb

```diff
@@ -1023,6 +1023,356 @@
     "  - [`CollabList`](/collab.html#CollabList) for collaborative filtering"
```

Comment:
<p>The command `repr(il_data)` calls `__repr__` under the hood. We don't need to document this here as this is standard python. I would just show the `il.data` object and live it at that.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/2026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
```suggestion
    "  - [`CollabList`](/collab.html#CollabList) for collaborative filtering"
```

Reply:
I see thanks!